webapp/maps_heatmaps/src/queryDB.ts

```diff
@@ -16,56 +17,57 @@
 
 import { database } from "./index";
 import * as firebase from "firebase";
-import * as geofirestore from "geofirestore";
 import { DateTime } from "./interface";
 
-/* @param center The center of the current map, 
-   @param radius The radius of the circle that contains the current map bounderies
-   @param labels The labels the client queries by
-   @param year, month, day The date the client queries by
+const databaseCollection = database.collection("Images");
+
+/* @param labels The labels the client queries by
+   @param datetime The date the client queries by
+   @param hash The hash of the current map bounderies
    @return The filtered collection by the different queries*/
 function getQueriedCollection(
```

Comment:
Add a docstring about what this functions does exactly.

Reply:
done, add a docstring.